### ISSUE TITLE
Fitbit software error with Glance settings

### ISSUE THREAD

**Issue by paulheaton81** [Rytiggy/Glance#96]
Hi, I have recently purchased a Fitbit Versa 2. I am running xdrip on a galaxy s8 using a freestyle libre and miaomiao senior to relay cgm data. I installed the glance watch face and altered xdrips settings as per setup instructions. When I change the settings on the glance software so that the data source is xdrip the Fitbit app closes and errors when a sync occurs making it very difficult to get back into the Fitbit app to switch the feature off so that it works again! Can anybody help? Has anybody had this issue?

**Comment by Rytiggy** [comment]
Hmm that sounds very strange, have you tried uninstalling the Fitbit app and reinstalling it?

**Comment by paulheaton81** [comment]
Yes I have, as soon as I select xdrip as a data source the app syncs with my watch and Fitbit app closes with errors.

**Comment by Rytiggy** [comment]
Could you show me the errors?

**Comment by paulheaton81** [comment]
It's just the general Android app keeps closing error that I get. If there's a way I can post a screen video capture I could do that?

**Comment by paulheaton81** [comment]
I will try grab some screenshots of the error.

**Comment by paulheaton81** [comment]
Typical! I have just re-installed everything including an older version of xdrip and it now works! Really sorry for wasting your time, I can't even replicate what happened in the first place to help you with testing for future problems. Hopefully it remains stable! Thanks again :)
